NedyUdombat/ReachHigh
# Setup tests and travis CI

#### What does this PR do?
Setup tests and travis CI

#### Description of Task to be completed?
- [x] Install required dependencies
- [x] Setup Travis-CI and add a badge to README.md
- [x] Write test for entry endpoint

#### How should this be manually tested?
* clone this repo
* checkout to `ch-setup-test-travis-169169219` branch
* run `npm install`
* run `npm t`
* You see the test run with a 100% test coverage

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?(if applicable)
[Pivotal tracker ID](https://www.pivotaltracker.com/story/show/169169219)

#### Screenshots
